xonsh/ptk/shortcuts.py

```diff
@@ -64,6 +64,8 @@ def prompt(self, message='', **kwargs):
 
         # Create CommandLineInterface.
         if self.cli is None:
+            if self.major_minor < (0, 57):
+                kwargs.pop('reserve_space_for_menu', None)
```

Comment:
The PTK change log says it was introduced in 0.55. But I guess it shouldn't matter. Else this PR seems fine to me. @scopatz Should wait for response on the issue before we merge?